cdx-cli/db-setup/jit-workload-account/7.update-task-definition.sh

```diff
@@ -0,0 +1,200 @@
+#!/bin/bash
+set -e
+set -u
+
+# Function for logging
+log() {
+    echo "[$(date +'%Y-%m-%dT%H:%M:%S%z')]: $*"
+}
+
+prompt_with_default() {
+    local prompt="$1"
+    local default_value="$2"
+    read -p "$prompt [$default_value]: " user_input
+    echo "${user_input:-$default_value}"
+}
+
+# Set static variables
+AWS_REGION="ap-south-1"
+ACCOUNT_ID=$(aws sts get-caller-identity --query "Account" --output text)
+OLD_BUCKET="jit-db-20250121011739"
+NEW_BUCKET=$(prompt_with_default "Enter S3 Bucket Name" "cdx-jit-db-logs-mv-innovation" )
+SECRET_NAME="CDX_SECRETS"
```

Comment:
<b>Secret:</b> Secret Keyword <br/>
                    <b><a href="https://console.cloudanix.com/code/findings/d803ac66c50fdc16/PR/1/artifacts/Divyansh/latest/secrets/show" target='_blank'>Click here for more details</a></b>